client/src/client.js

```diff
@@ -39,7 +39,7 @@ class Client {
   }
 
   bootstrap() {
-    const endpoint = process.env.EVERGREEN_ENDPOINT;
+    const endpoint = this.removeTrailingSlashes(process.env.EVERGREEN_ENDPOINT);
```

Comment:
I know why this is here, and [I'm going to fix this on the server](https://issues.jenkins-ci.org/browse/JENKINS-51113). Redundant slashes shouldn't be an issue.